eos/constraints/B-to-Kstar-ll.yaml

```diff
@@ -1341,72 +1341,72 @@ B^0->K^*0mu^+mu^-::NormalizedBR[0.10,0.98]@LHCb:2016A:
     observable: B->K^*ll::NormalizedBR
     kinematics: {q2_min: 0.1, q2_max: 0.98}
     options: {l: mu, q: d, psi: "J/psi", tag: BFS2004}
-    mean: 8.54e-5
```

Comment:
Can you add a bit of text to the commit, explaining that you changed the data from the ``q2``-averaged branching ratio to the `´q2``-integrated branching ratio?